src/components/ContactUs/Notification.js

```diff
@@ -0,0 +1,83 @@
+// eslint-disable-next-line react/prop-types
```

Comment:
No exceptions please, you can add prop types to your props so you  wont have to add an exception like this. May google be on your side

Reply:
Done. There will be no more eslint exceptions anymore =)